packages/components/src/internal/components/domainproperties/AdvancedSettings.tsx

```diff
@@ -349,10 +352,15 @@ export class AdvancedSettings extends React.PureComponent<AdvancedSettingsProps,
                             id={createFormInputId(DOMAIN_FIELD_PHI, domainIndex, index)}
                             onChange={this.handleChange}
                             value={PHI}
-                            disabled={domainFormDisplayOptions.phiLevelDisabled || field.disablePhiLevel}
+                            disabled={disablePhiSelect}
                         >
-                            {phiLevels.map((level, i) => (
-                                <option key={i} value={level.value}>
+                            {!currentValueExists && (
+                                <option key={PHI} value={PHI}>
+                                    {DOMAIN_PHI_LEVELS.find(level => level.value === PHI)?.label ?? PHI}
+                                </option>
+                            )}
+                            {phiLevels.map(level => (
```

Comment:
Do we still need all the options if the select is disabled?

Reply:
Nope, we don't need them in that case....but they also aren't hurting anything and there aren't very many of them, so I will likely leave it as is for now.